build/ci/evergreen.yml

```diff
@@ -1202,7 +1202,7 @@ buildvariants:
     run_on:
       - rhel70-small
     expansions:
-      server_version: "4.2.2"
+      server_version: "4.2.3"
```

Comment:
prep for the next release

Reply:
This looks to be getting to the point where there's a number of steps to do for each release that we should track in a doc or .MD somewhere.

Reply:
Updated RELEASING.md